### ISSUE TITLE
Fix note-dropping bug

### ISSUE THREAD

**Issue by SuperDisk** [SuperDisk/UGE#145]
This one is long-standing, really annoying, and should be prioritized.

> Q: Why is hT with hD dropping a lot of pulse and noise note-ons on playback?

**Comment by 272354** [comment]
At least for me, this is still an issue in my current version. Notes are still sometimes dropped in playback, and this carries over into rendered audio files. It seems to happen frequently enough that it's rare to get a recording with no dropped notes at all. Does this happen on your end?

**Comment by SuperDisk** [comment]
Yeah, it does and it's not good. Unfortunately I haven't had time lately to sit down and squash bugs, but I welcome anyone else to look into this one as it's been around since the first version.

**Comment by nikku4211** [comment]
I don't know if you've noticed this before, but not only does it _drop_ notes, it also plays notes that aren't supposed to exist anymore, usually after I've deleted them. And it takes a while to try to get the tracker used to that note not being there anymore.

**Comment by SuperDisk** [comment]
@nikku4211 I think that's a different bug which is very fickle for me to reproduce. As for why it happens... it's actually quite baffling, since the tracker grid is painted directly based on the source data, so I'm not sure how it gets stale.
